### ISSUE TITLE
Accessibility: `ALT+0` help menu doesn't work

### ISSUE THREAD

**Issue by GuySartorelli** [silverstripe/silverstripe-htmleditor-tinymce#25]
According to https://www.tiny.cloud/docs/tinymce/6/tinymce-and-screenreaders/ and read out to (some) screen readers, pressing `ALT+0` should display a help menu. This works as expected in [the examples in tinymce website](https://www.tiny.cloud/docs/tinymce/6/basic-example/).

Without this, keyboard users have no way of knowing how to access the menu items.

## Acceptance Criteria

- [x] When focus is inside the TinyMCE editor, pressing `ALT+0` brings up the accessibility menu
- [x] There is some visual indicator for sighted keyboard users that pressing `ALT+0` will open this menu OR a link to https://www.tiny.cloud/docs/tinymce/6/tinymce-and-screenreaders/ is provided

## PRs

- https://github.com/silverstripe/silverstripe-htmleditor-tinymce/pull/33
- https://github.com/silverstripe/developer-docs/pull/840
